apis/beacon/blocks/attestations.yaml

```diff
@@ -13,6 +13,9 @@ get:
   responses:
     "200":
       description: Success
+      headers:
+        Eth-Consensus-Version:
```

Comment:
I think we have to bump all apis to v2 since we now require the version header to be set

Reply:
I thought adding a header won't break backward compatibility but I'm not against bumping this API to v2 too since we're doing that for the other related ones

Reply:
Bumped up both APIs to V2 since the version is required in the data and that would be a breaking change